bitnami/zookeeper/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if gt .Values.replicaCount 1.0 -}}
```

Comment:
just curiosity, why `1.0` and not `1` ?

Reply:
It seems that either yaml or Helm is setting Values.replicaCount to float. And when using `1` the `gt` function gives the error that it can't compare two different types. 
I actually fixed it by following this: https://stackoverflow.com/questions/46009933/kubernetes-helm-chart-if-condition-check